### ISSUE TITLE
New placeholder request

### ISSUE THREAD

**Issue by M4RC0Sx** [IntellectualSites/PlotSquaredSuggestions#138]
**PlotSquared Feature Suggestion:**
Placeholder to get likes and dislikes of the plots.

**How should it be implemented?**
It should be compatible with PAPI and LeaderHeads, in order to create tops with the best and worst rated plots.

**Additional context**
Nothing.

**Checklist**:

- [X] I made sure there are no duplicates of my suggestion [(Use search)](https://github.com/IntellectualSites/PlotSquaredSuggestions/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+)
- [X] I made sure my suggestion does not exist yet [(Check the wiki)](https://wiki.intellectualsites.com/plotsquared/home)
- [X] I made sure my suggestion is useful for the majority of PlotSquared users

**Comment by RedstoneFuture** [comment]
`%plotsquared_currentplot_rating%`

**Comment by M4RC0Sx** [comment]
Yes, 2 separate placeholders, they should be by player I think, counting the total amount of likes and dislikes in all of a player´s plots. By doing that, it will work with leaderheads and PAPI, allowing us to create a leaderboard with the player with the biggest amount of likes and dislikes.
